l1-contracts/test/foundry/l1/integration/DeploymentTest.t.sol

```diff
@@ -69,45 +73,102 @@ contract DeploymentTests is L1ContractDeployer, ZKChainDeployer, TokenDeployer,
     // equals the balance of L1Shared bridge.
     function test_initialDeployment() public {
         uint256 chainId = zkChainIds[0];
-        IBridgehub bridgehub = IBridgehub(l1Script.getBridgehubProxyAddress());
-        address newChainAddress = bridgehub.getZKChain(chainId);
-        address admin = IZKChain(bridgehub.getZKChain(chainId)).getAdmin();
-        IChainTypeManager ctm = IChainTypeManager(bridgehub.chainTypeManager(chainId));
+        address newChainAddress = bridgeHub.getZKChain(chainId);
+        address admin = IZKChain(bridgeHub.getZKChain(chainId)).getAdmin();
 
         assertNotEq(admin, address(0));
         assertNotEq(newChainAddress, address(0));
 
-        address[] memory chainAddresses = bridgehub.getAllZKChains();
+        address[] memory chainAddresses = bridgeHub.getAllZKChains();
         assertEq(chainAddresses.length, 1);
         assertEq(chainAddresses[0], newChainAddress);
 
-        uint256[] memory chainIds = bridgehub.getAllZKChainChainIDs();
+        uint256[] memory chainIds = bridgeHub.getAllZKChainChainIDs();
         assertEq(chainIds.length, 1);
         assertEq(chainIds[0], chainId);
 
-        uint256 protocolVersion = ctm.getProtocolVersion(chainId);
-        assertEq(protocolVersion, 0);
+        uint256 protocolVersion = chainTypeManager.getProtocolVersion(chainId);
+        assertEq(protocolVersion, 25);
     }
 
-    function test_bridgehubSetter() public {
-        IBridgehub bridgehub = IBridgehub(l1Script.getBridgehubProxyAddress());
+    function test_bridgeHubSetter() public {
         uint256 chainId = zkChainIds[0];
-        IChainTypeManager chainTypeManager = IChainTypeManager(bridgehub.chainTypeManager(chainId));
         uint256 randomChainId = 123456;
 
         vm.mockCall(
             address(chainTypeManager),
             abi.encodeWithSelector(IChainTypeManager.getZKChainLegacy.selector, randomChainId),
             abi.encode(address(0x01))
         );
-        vm.store(address(bridgehub), keccak256(abi.encode(randomChainId, 205)), bytes32(uint256(uint160(1))));
+        vm.store(address(bridgeHub), keccak256(abi.encode(randomChainId, 205)), bytes32(uint256(uint160(1))));
         vm.store(
-            address(bridgehub),
+            address(bridgeHub),
```

Comment:
the previous naming is more correct, our contract is named `Bridgehub` and so its small casing is `bridgehub`